src/gui/optionsdlg.cpp

```diff
@@ -425,6 +426,13 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     m_ui->textTempPath->setDialogCaption(tr("Choose a save directory"));
     m_ui->textTempPath->setMode(FileSystemPathEdit::Mode::DirectorySave);
 
+    // disable mouse wheel event on widgets to avoid mis-selection
+    Utils::Gui::WheelEventEater *wheelEventEater = new Utils::Gui::WheelEventEater(this);
+    for (const auto &widget : copyAsConst(findChildren<QComboBox *>()))
```

Comment:
Isn't it overthinking to declare it as reference in this case? Would that lead to a double dereference behind the scenes?